src/sql/base/browser/ui/card/card.ts

```diff
@@ -0,0 +1,176 @@
+import 'vs/css!./media/card';
+import * as DOM from 'vs/base/browser/dom';
+import { IColorTheme, ICssStyleCollector, registerThemingParticipant } from 'vs/platform/theme/common/themeService';
+import { inputBorder, inputValidationInfoBorder } from 'vs/platform/theme/common/colorRegistry';
+import { calloutDialogBodyBackground } from 'sql/platform/theme/common/colorRegistry';
+import { Widget } from 'vs/base/browser/ui/widget';
+import { Event, Emitter } from 'vs/base/common/event';
+
+export class Card extends Widget {
```

Comment:
We already have a card component. So you should either use this there or figure out a different name to avoid confusion.